hosting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Hosting
+layout: recommendation
+expires: 2017-11-01
+---
+
+GDS follows the [Government Cloud First policy](https://www.gov.uk/guidance/government-cloud-first-policy) meaning that we prefer the use of PaaS and IaaS solutions over managing our own hardware
+
+## User Needs
+
+* Users want systems that are highly scalable and available
+* GDS wants to manage the environments easily and simply, with confidence via automated tools
+
+## Principles
+
+GDS will build new tools on only one cloud platform for each of IaaS and PaaS.
+
+We prefer systems built on a PaaS to systems built on IaaS where possible.
+
+We prefer cloud suppliers that are well supported by common tooling to reduce cost of migrating to a new supplier.
+
+## Tools
+
+GDS has selected two cloud platforms:
+
+### PaaS
+
+* GOV.UK PaaS
+
+### IaaS
+
+* Amazon Web Services
```

Comment:
Add a note about what AWS products we are selecting.  We are aiming for minimal IaaS tools, rather than the value add tools, which helps us reduce lockin